pkg/trace/api/normalizer_test.go

```diff
@@ -430,6 +431,7 @@ func TestNormalizeTag(t *testing.T) {
 		{in: "test\x99\x8faaa", out: "test_aaa"},
 		{in: "test\x99\x8f", out: "test"},
 		{in: strings.Repeat("a", 888), out: strings.Repeat("a", 200)},
+		{in: strings.Repeat("a", 888), out: strings.Repeat("a", 200)},
```

Comment:
Looks like a duplicate test case.

Reply:
I reviewed my own PR and yet I've missed this. Thanks. Removed.